skbio/stats/tests/test_power.py

```diff
@@ -134,6 +142,24 @@ def test_subsample_power(self):
         self.assertEqual(test_p.shape, (2, 5))
         npt.assert_array_equal(np.arange(5, 50, 10), test_c)
 
+    def test_subsample_power_matches(self):
+        test_p, test_c = subsample_power(self.f,
+                                         samples=self.pop,
+                                         num_iter=10,
+                                         num_runs=5,
+                                         draw_mode="matched")
+        self.assertEqual(test_p.shape, (5, 4))
+        npt.assert_array_equal(np.array([10, 20, 30, 40]), test_c)
+
+    def test_subsample_power_multi_p(self):
+        f = lambda x: np.array([0.5, 0.5])
```

Comment:
If I remember correctly, pep8 hates the old lambda definition.  Use def f(x): return np.array([0.5, 0.5])

Here's the [pep8 reference](https://www.python.org/dev/peps/pep-0008/)